<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>698</width>
    <height>560</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Task Scheduler (Liu algorithm)</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="table">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>140</width>
      <height>506</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOn</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="tabKeyNavigation">
     <bool>false</bool>
    </property>
    <property name="showGrid">
     <bool>true</bool>
    </property>
    <property name="sortingEnabled">
     <bool>false</bool>
    </property>
    <attribute name="horizontalHeaderVisible">
     <bool>true</bool>
    </attribute>
    <attribute name="horizontalHeaderMinimumSectionSize">
     <number>30</number>
    </attribute>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>30</number>
    </attribute>
    <attribute name="horizontalHeaderStretchLastSection">
     <bool>true</bool>
    </attribute>
    <attribute name="verticalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderMinimumSectionSize">
     <number>24</number>
    </attribute>
    <attribute name="verticalHeaderDefaultSectionSize">
     <number>24</number>
    </attribute>
    <column>
     <property name="text">
      <string>#</string>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </column>
    <column>
     <property name="text">
      <string>p</string>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </column>
    <column>
     <property name="text">
      <string>r</string>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </column>
    <column>
     <property name="text">
      <string>d</string>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </column>
   </widget>
   <widget class="QPlainTextEdit" name="console">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>40</y>
      <width>360</width>
      <height>471</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <family>Consolas</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOn</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_schedule">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>522</y>
      <width>361</width>
      <height>24</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Run scheduler</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_randomize">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>40</y>
      <width>120</width>
      <height>24</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Randomize</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_tasks">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Tasks</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_console">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>20</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Console</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_add">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>130</y>
      <width>120</width>
      <height>24</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string>Add</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_edit">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>160</y>
      <width>120</width>
      <height>24</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string>Edit</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_delete">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>190</y>
      <width>120</width>
      <height>24</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string>Delete</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_clear">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>70</y>
      <width>120</width>
      <height>24</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string>Clear</string>
    </property>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>table</tabstop>
  <tabstop>btn_randomize</tabstop>
  <tabstop>btn_clear</tabstop>
  <tabstop>btn_add</tabstop>
  <tabstop>btn_edit</tabstop>
  <tabstop>btn_delete</tabstop>
  <tabstop>console</tabstop>
  <tabstop>btn_schedule</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
